modules/@apostrophecms/asset/lib/globalIcons.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
   'plus-icon': 'Plus',
   'phone-icon': 'Phone',
   'refresh-icon': 'Refresh',
+  'unfold-more-horizontal-icon': 'UnfoldMoreHorizontal',
```

Comment:
we already have something that does `unfold-more-horizontal-icon` .. we use it in the pagination UI to explicitly open the page numbers.

That said, I like the way this one is spaced more, can we replace that one with this one?